DecodeCx1.py

```diff
@@ -5,195 +5,160 @@
 
 
 tester_details=[]
-r=''  # To save the value of block_insights response
+n1=9  # No. of callings for all blocks
+n2=2  # No. of testers giving test
+n3=8  # No. of block_insights

```

Comment:
number of block_insights should be equal to `number_of_blocks` right?

Reply:
actually no. of blocks are 9 including thankyou page. I am also calling the thankyou page.